librz/analysis/analysis.c

```diff
@@ -227,6 +228,14 @@ RZ_API bool rz_analysis_use(RzAnalysis *analysis, const char *name) {
 				return false;
 			}
 			rz_analysis_set_reg_profile(analysis);
+
+			// default : init and enable RZIL if defined rzil_init
+			if (h->rzil_init) {
+				rz_analysis_rzil_setup(analysis);
+			} else {
+				// create it to make analysis_tp go right
+				analysis->rzil = rz_analysis_rzil_new();
```

Comment:
This `if/else` == `if enable rzil` for now, because for now we both have ESIL and new IL in code. We also create an empty `rzil` to keep the logic as simple as possible, and make it not affect the original ESIL.